src/Orbit/main.cpp

```diff
@@ -375,7 +376,11 @@ int main(int argc, char* argv[]) {
   command_line_flags =
       orbit_command_line_utils::RemoveFlagsNotPassedToMainWindow(command_line_flags);
 
+  auto confidence_interval_estimator =
+      std::make_unique<const orbit_statistics::WilsonBinomialConfidenceIntervalEstimator>();
```

Comment:
So, the main question I have here is: Do we really want to create this in `main` and then pass it to `OrbitMainWindow` and then to `OrbitApp`? What about just making `OrbitApp` create and own it?
I'd also like to hear from the other reviewers on this.

Reply:
Hypothetically, in the future we might want to add a command line argument controlling the confidence level (e.g. 95% vs 99%) or turning the thing off altogether.

Reply:
As you say, this is hypothetical. It could be said of almost all features. Also: Imagine that instead of a command line option we added a UI option, then this would already be out of place. For me we should keep it simple and adjust on need.